src/scripts/store/dimStoreItem.directive.template.html

```diff
@@ -30,6 +30,7 @@
          width="44" />
   </div>
   <div ng-class="::vm.badgeClassNames"
+       class="{{ vm.item.dtrRating || vm.item.quality ? 'item-stat-no-bg':'' }}"
```

Comment:
This needs to be part of `badgeClassNames` - we can't just bind class to an expression here.

We'll probably need to include the dtr rating in the index string.